extensions/WEBGL_polygon_mode/extension.xml

```diff
@@ -19,6 +19,12 @@
     <mirrors href="https://chromium.googlesource.com/angle/angle/+/HEAD/extensions/ANGLE_polygon_mode.txt"
              name="ANGLE_polygon_mode">
     </mirrors>
+
+    <p>
+      <b>Note:</b> Browsers should should discourage use of this extension in non-debug applications
```

Comment:
Consider using `<div class="note">` instead of an inline label for consistency with other notes.

Reply:
Done.